notebook/static/notebook/js/outputarea.js

```diff
@@ -336,7 +336,7 @@ define([
             if (needs_height_reset) {
                 that.element.height('');
             }
-            that.element.trigger('resize');
+            that.element.trigger('resize.Output');
```

Comment:
We scope events to components when they are global events (e.g. on `events.trigger('event.Component')`). These are events on a specific element, so scoping seems redundant, doesn't it? For instance, you still can't listen for all `resize.Output` events with this, can you? Is that what you are after?